### PR TITLE
add podspec

### PR DESCRIPTION
**Pull request by carlosalmonte04**
Add podspec file

## Description

Adds podspec file to root, allowing this module to be managed by cocoapods

## Related Issue

https://github.com/adobe/react-native-acpcore/issues/2

## Motivation and Context

React Native projects that have all dependencies managed by cocoapods would break pattern to use this @adobe/react-native-acpcore

## How Has This Been Tested?

1. Add `pod 'RCTACPCore', :path => '../node_modules/@adobe/react-native-acpcore'` to your podfile
2. Run `pod install` in the `ios/` directory
3. Use module as usual on ios

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [  ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: RCTACPCore.podspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  s.license      = "Apache License"
+  s.platform     = :ios
+
+  s.source       = { :git => "https://github.com/adobe/react-native-acpcore.git", :tag => "#{s.version}" }
+
+  s.source_files  = "ios/**/*.{h,m}"
+  s.requires_arc = true
+
+  s.dependency "React"  
+  s.dependency "ACPGriffonBeta", "~> 0.0"
```
Comment (nporter-adbe) [inline RCTACPCore.podspec]: I don't believe all these dependencies are required for all use cases. It should probably only depend on React IMO.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,13 @@ public void onCreate() {
 ```
 
 #### 3.2 iOS project
+
+#### With Cocoapods
```
Comment (nporter-adbe) [inline README.md]: 👍

File: RCTACPCore.podspec (hunk 1 above)
Comment (carlosalmonte04) [reply RCTACPCore.podspec]: Thanks! True. Updated according, let me know if ok. - Do we need the frameworks and libraries?

**Comment by nporter-adbe** [comment]
@carlosalmonte04 Thanks for your contribution! I'll get this merged in very soon, in the meantime I'm going to try and get this change into our other extensions (Analytics, Audience, Campaign, Target, and User Profile). If you're interested in making PR's in the other repos with this change feel free.

File: RCTACPCore.podspec (hunk 1 above)
Comment (shalehaha) [reply RCTACPCore.podspec]: I believe we need to link all the libraries inside https://github.com/adobe/react-native-acpcore/tree/master/ios/libs

File: README.md (hunk 2 above)
Comment (booker-dragon) [reply README.md]: I think `react-native link` will automatically add this to Podfile

https://facebook.github.io/react-native/docs/linking-libraries-ios#step-2

**Comment by nporter-adbe** [comment]
Closing in favor of https://github.com/adobe/react-native-acpcore/pull/32
Thanks for getting this started.

**Pull request closed without merging**